src/NerdBank.GitVersioning/version.schema.json

```
@@ -188,13 +188,19 @@
               "default": "minor"
             },
             "firstUnstableTag": {
-              "description": "Specifies the first/default prerelease tag for new versions",
+              "description": "Specifies the first/default prerelease tag for new versions.",
               "type": "string",
               "default": "alpha"
             }
           },
           "additionalProperties": false
         },
+        "tagName": {
+          "description": "Defines the format of tag names. Format must include a placeholder '{version}' for the version.",
+          "type": "string",
+          "pattern": ".*\\{version\\}.*",
```

Nit: you can drop the wildcards on either side because you're not using `^` and `$` anyway, so a match is already assumed to be somewhere in the middle.
```suggestion
          "pattern": "\\{version\\}",
```

---

I copied this pattern from the `branchName` property, but I'll change both to use the less verbose syntax 👍🏻